bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -61,11 +61,11 @@
 
 	<!-- Color Temperature -->
 	<channel-type id="colorTemperature">
-		<item-type>Number</item-type>
+		<item-type unitHint="K">Number:Temperature</item-type>
 		<label>Color Temperature</label>
 		<description>This channel supports adjusting the color temperature from 2200K to 6500K.</description>
 		<category>LightBulb</category>
-		<state min="2200" max="6500" pattern="%d K"/>
+		<state step="100" min="2200" max="6500" pattern="%.0f K"/>
```

Comment:
Readme states min=2500

Reply:
What do you suggest? Either the read me or the prior code is wrong. I did not change that. Probably keeping 2200 K is less likely to upset anyone.

Reply:
Maybe @wildcs knows the right value?